web/src/project/ProjectSummary.tsx

```diff
@@ -12,6 +12,14 @@ import { Input, Button } from 'reactstrap'
 
 const PAGE_SIZES = [20, 40, 100, 1000]
 
+const sanitiseValue = (value: any) => {
+    const tvalue = typeof value
+    if (tvalue === 'string' || tvalue === 'number') return value
```

Comment:
Returns a number or string, this is correct? All other types seem to be stringified.